build.sbt

```diff
@@ -827,7 +827,7 @@ lazy val baseEngineComponents = (project in engine("base/components")).
   settings(assemblySampleSettings("baseEngineComponents.jar"): _*).
   settings(
     name := "nussknacker-baseengine-components",
-  ).dependsOn(baseEngineApi)
+  ).dependsOn(baseEngineApi, avroUtil)
```

Comment:
doesn't seem to be needed? :)

Reply:
it is part of longer story :D (forgot to setup Draft)